apps/re/lib/partner_brokers/partner_brokers.ex

```
@@ -31,4 +39,28 @@ defmodule Re.PartnerBrokers do
       "uuid" => uuid
     })
   end
+
+  def create_account(uuid) do
+    case Users.get_preloaded(uuid, :districts) do
+      {:ok, user} -> user |> Salesforce.create_account() |> handle_job_error()
+      error -> handle_job_error(error)
+    end
+  end
+
+  def update_account(uuid) do
+    case Users.get_preloaded(uuid, :districts) do
+      {:ok, user} -> user |> Salesforce.update_account() |> handle_job_error()
+      error -> handle_job_error(error)
+    end
+  end
+
+  defp handle_job_error({:ok, result}), do: {:ok, result}
```

These functions should be in the `JobQueue` module and no function on this module should raise an exception since we're raising only because of `JobQueue`.